src/Functions/tokens.cpp

```diff
@@ -26,7 +26,7 @@ class FunctionTokens : public IFunction
 public:
     static constexpr auto name = "tokens";
 
-    static FunctionPtr create(ContextPtr) { return std::make_shared<FunctionTokens>(); }
+    static FunctionPtr create(ContextPtr) { return std::make_shared<FunctionTokensExtractor>(); }
```

Comment:
Could we make the sparse ngrams tokenizer also available at SQL level via this function, see l. 75-89 below.